### PR TITLE
Set up application structure for Client

### PR DESCRIPTION
**Pull request by robin-lillqvist**
# [PT Story](https://www.pivotaltracker.com/story/show/171561376)
```
As a development team
In order to be able to start adding features
We would like to have an basic application structure
```
## Changes proposed in this pull request:
* Scaffold a React application using create-react-app
* Clean up unnecessary code from scaffold

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (emtalen) [inline src/App.test.js]: Delete this file?

**Review by emtalen — CHANGES_REQUESTED** [review]

File: src/App.test.js (hunk 1 above)
Comment (robin-lillqvist) [reply src/App.test.js]: Done

**Review by emtalen — APPROVED** [review]

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (tochman) [inline src/App.css]: No need to keep this around

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
```
Comment (tochman) [inline src/App.jsx]: Use fragments and skip the className

File: src/index.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
```
Comment (tochman) [inline src/index.css]: No need to keep this

**Review by tochman — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 2 above)
Comment (robin-lillqvist) [reply src/App.css]: Done

File: src/App.jsx (hunk 3 above)
Comment (robin-lillqvist) [reply src/App.jsx]: Done

File: src/index.css (hunk 4 above)
Comment (robin-lillqvist) [reply src/index.css]: Done

**Comment by tochman** [comment]
Please clean up the PR description from unused sections.

**Comment by robin-lillqvist** [comment]
I pruned it again now, @tochman. Hope that it can be merged this time.

**Comment by robin-lillqvist** [comment]
removed some css file dependencies

**Pull request merged**